kubernetes/linux/Dockerfile.multiarch

```diff
@@ -35,5 +35,12 @@ ENV AGENT_VERSION ${IMAGE_TAG}
 WORKDIR ${tmpdir}
 
 RUN chmod 775 $tmpdir/*.sh; sync; $tmpdir/setup.sh ${TARGETARCH}
-CMD [ "/opt/main.sh" ]
 
+# Do vulnerability scan in a seperate stage to avoid adding layer
+FROM base_image AS vulnscan
+COPY --from=aquasec/trivy:latest /usr/local/bin/trivy /usr/local/bin/trivy
+RUN trivy rootfs --exit-code 1 --ignore-unfixed --no-progress --severity HIGH,CRITICAL --skip-files "/usr/sbin/telegraf" --skip-files "/opt/telegraf" --skip-files "/usr/local/bin/trivy" /

```

Comment:
This will fail the build in azure pipelines if any vulnerabilities are found, right?

Reply:
yes